### PR TITLE
Augment testcase names to include type and category.

### PR DESCRIPTION
**Pull request by ralphbean**
At the request of @thrix.

### CODE REVIEW

**Comment by ralphbean** [comment]
@thrix, is this how you'd like it to be?

File: resultsdbupdater/utils.py [hunk 1]
```diff
@@ -191,8 +192,17 @@ def _construct_testcase_dict(msg):
     # Invert jenkins.osci into osci.jenkins
     namespace = '.'.join(reversed(namespace.split('.')))
 
+    # These fields will definitely be a part of the testcase name.
+    tokens = [namespace, job]
+
+    # Optionally add these results if present
```
Comment (mprahl) [inline resultsdbupdater/utils.py]: "results" here is misleading. How about "values"?

File: resultsdbupdater/utils.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
 import uuid
 import re
 
+import fedmsg
 import requests
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
-import fedmsg
+import six
```
Comment (mprahl) [inline resultsdbupdater/utils.py]: Can you add `six` to `requirements.txt`?

**Comment by mprahl** [comment]
:+1:

**Pull request merged**